### ISSUE TITLE
Validation 

### ISSUE THREAD

**Issue by hlaahmed** [autonomousvision/transfuser#39]
Hello, I was training a model I wrote using your 14 weather minimal dataset
The training loss decreased through epochs
But validation loss kept decreasing and increasing 
It ranged from 0.26 to 0.38
Did the same happen to you ? And does that mean that the model is not learning?

**Comment by ap229997** [comment]
Hi, sorry for the late reply.

It is normal for the validation loss to fluctuate during training. You should look at the overall trend and that should be decreasing. We also used early stopping where we stopped the training if the validation loss did not improve for 25 epochs (we validate after every 5 epochs).
